index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>ILC 5</title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css">
+  <link rel="stylesheet" href="css/style.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
+  <script src="js/accounting.min.js"></script>
+  <script src="js/script.js"></script>
+</head>
+    <!--
+    <a id="get-employees" href="#">Click me to get employees</a>
+    -->
+    <button id='get-employees' class='centered'>Click me to get employees</button>
+    <div id="loader" style="display:none;" class='centered'><image src="img/gif-load.gif"/></div>
+    <div id="holder"></div>
+    <div id="dialog" title=""></div>
```

Comment:
Great! Appropriate script and css links! all the needed divs! button in jQuery! 👍